### ISSUE TITLE
error: unexpected ar::Type in add_bitcast()

### ISSUE THREAD

**Issue by practicalswift** [NASA-SW-VnV/ikos#64]
```
$ git clone https://github.com/sipa/minisketch minisketch-ikos
$ cd minisketch-ikos/src/
$ ikos-scan make
…
Analyze test-exhaust? [Y/n]
[*] Running ikos test-exhaust.bc -o test-exhaust.db
[*] Running ikos preprocessor
[*] Running ikos analyzer
[!] ikos was compiled in debug mode, the analysis might be slow
[*] Translating LLVM bitcode to AR
ikos-analyzer: /tmp/ikos-SS01Rz/test-exhaust.pp.bc: error: unexpected ar::Type in add_bitcast()
ikos: error: a run-time error occurred
```

**Comment by arthaud** [comment]
Thanks for reporting it. This is fixed by ab441860ea2a99582324b71896163327d7a77e00.

**Comment by practicalswift** [comment]
Confirming that I'm no longer hitting this issue. Thanks for fixing!
